sdk/python/kfp_tekton/compiler/compiler.py

```diff
@@ -627,6 +628,7 @@ def _write_workflow(workflow: List[Dict[Text, Any]],  # Tekton change, signature
       package_path: file path to be written. If not specified, a yaml_text string
         will be returned.
     """
+    # yaml_text = dump_yaml(workflow)
```

Comment:
KFP changed how they `dump` their YAML in `kfp.components._yaml_utils` but more work would be needed to follow a similar approach for Tekton YAML